# in exercise 5a. x = 0 because the other x
# is in an inner scope within a iteration command
# in exercise 5b. there is an error because we can't access the inner x because its within a do/end block.
